src/docs/OAuth2.yaml

```diff
@@ -0,0 +1,10 @@
+paths:
+  /api/auth/google:
+    get:
+      tags:
+      - "OAuth2"
+      summary: "Authenticates user using google 3rd party authentication service"
+      responses:
+        "200":
+          description: "Redirects verified userس to the home page"
```

Comment:
I am not sure if that is a typo but you added an arabic character after user in line 9